system/include/emscripten/wasm_worker.h

```diff
@@ -389,6 +343,15 @@ ATOMICS_WAIT_TOKEN_T emscripten_condvar_wait_async(emscripten_condvar_t *condvar
 // ("broadcast" operation).
 void emscripten_condvar_signal(emscripten_condvar_t *condvar __attribute__((nonnull)), int64_t numWaitersToSignal);
 
+// Legacy names for emscripten_atomic_wait/notify functions, defined in
+// emscripten/atomic.h
+#define emscripten_wasm_wait_i32 emscripten_atomic_wait_u32
+#define emscripten_wasm_wait_i64 emscripten_atomic_wait_u64
+#define emscripten_wasm_notify emscripten_atomic_notify
+#pragma clang deprecated(emscripten_wasm_wait_i32, "use emscripten_atomic_wait_u32 instead")
+#pragma clang deprecated(emscripten_wasm_wait_i64, "use emscripten_atomic_wait_u64 instead")
+#pragma clang deprecated(emscripten_wasm_notify, "use emscripten_atomic_notify instead")
+
```

Comment:
@tlively what do you think about the naming of these macros.. should we use `i32`/`i64` to match llvm types or should we follow all the existing `atomic` functions in `atomic.h` which have names like `emscripten_atomic_exchange_u8` and `emscripten_atomic_cas_u8`.

Reply:
Or I guess we could side step the issue and use `emscripten_atomic_wait32` and `emscripten_atomic_wait64`which matches the names of the llvm builtins?

Reply:
Using `...wait_u32` and `...wait_u64` to be consistent with other names using `..._u8`, etc. sounds good to me.